common/src/lib.rs

```diff
@@ -419,6 +419,19 @@ pub fn code_exists(code_hash: H256) -> bool {
     sp_io::storage::exists(&code_key(code_hash, CodeKeyPrefixKind::RawCode))
 }
 
+pub fn reset_storage() {
+    sp_io::storage::clear_prefix(STORAGE_PROGRAM_PREFIX, None);
+    sp_io::storage::clear_prefix(STORAGE_PROGRAM_PAGES_PREFIX, None);
+    sp_io::storage::clear_prefix(STORAGE_MESSAGE_PREFIX, None);
+    sp_io::storage::clear_prefix(STORAGE_MESSAGE_NONCE_KEY, None);
+    sp_io::storage::clear_prefix(STORAGE_MESSAGE_USER_NONCE_KEY, None);
+    sp_io::storage::clear_prefix(STORAGE_CODE_PREFIX, None);
+    sp_io::storage::clear_prefix(STORAGE_CODE_METADATA_PREFIX, None);
+    sp_io::storage::clear_prefix(STORAGE_CODE_REFS_PREFIX, None);
+    sp_io::storage::clear_prefix(STORAGE_WAITLIST_PREFIX, None);
+    sp_io::storage::clear_prefix(GAS_VALUE_PREFIX, None);
```

Comment:
Code prefix includes metadata and refs, and message prefix includes nonce.

```suggestion
    sp_io::storage::clear_prefix(STORAGE_PROGRAM_PREFIX, None);
    sp_io::storage::clear_prefix(STORAGE_PROGRAM_PAGES_PREFIX, None);
    sp_io::storage::clear_prefix(STORAGE_MESSAGE_PREFIX, None);
    sp_io::storage::clear_prefix(STORAGE_CODE_PREFIX, None);
    sp_io::storage::clear_prefix(STORAGE_WAITLIST_PREFIX, None);
    sp_io::storage::clear_prefix(GAS_VALUE_PREFIX, None);
```

Alternatively we can clear `g::` prefix at once but it is quite dangerous.

Reply:
anything can be stored under `g::` prefix, so this is definitely not suitable